app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/DeviceCommunicationService.java

```diff
@@ -282,36 +183,29 @@ public synchronized int onStartCommand(Intent intent, int flags, int startId) {
 
         Prefs prefs = getPrefs();
         switch (action) {
-            case ACTION_START:
+/*            case ACTION_START: {
                 start();
                 break;
```

Comment:
Why did you comment this out?

Reply:
Should've clarified (and will do) that the action is handled at line 151.